main/com/domain/LadderLine.java

```diff
@@ -0,0 +1,36 @@
+package main.com.domain;
+
+import java.util.Random;
+
+public class LadderLine {
+	private static final	Random random = new Random();
```

Comment:
```suggestion
	private static final Random random = new Random();
```
왜 `Random`을 상수로 관리해주셨나요?